manifests/init.pp

```diff
@@ -1,5 +1,30 @@
+# == Class: dns
+#
 # Install, configure and start dns service
-class dns(
+#
+# === Parameters:
+#
+# $acls::                  Specify a hash of ACLs. Each key is the
+#                          name of a network, and its value is
+#                          an array of subnet strings.
+#                          type:hash
+#
+# $controls::              Specify a hash of controls. Each key is the
+#                          name of a network, and its value is a hash
+#                          containing 'port' => integer, 'keys' => array
+#                          and 'allowed_addresses' => array
+#                          type:hash
+#
+# === Usage:
+#
+# * Simple usage:
+#
+#     include dns
+#
+# * Advanced usage:
```

Comment:
If you include the example here (or remove the Advanced usage:) it's good to go.